docs/source/index.rst

```diff
@@ -3,24 +3,24 @@
 Converting notebooks to other formats
 =====================================
 
-Newly added in the 1.0 release of IPython is the ``nbconvert`` tool, which 
-allows you to convert an ``.ipynb`` notebook document file into various static 
-formats. 
+Newly added in the 1.0 release of IPython is the ``nbconvert`` tool, which
+allows you to convert an ``.ipynb`` notebook document file into various static
+formats.
 
-Currently, ``nbconvert`` is provided as a command line tool, run as a script 
-using IPython. A direct export capability from within the 
-IPython Notebook web app is planned. 
```

Comment:
I don't care here, but for future reference, it's a good idea to disable stripping whitespace on save in editors. It's a horrible misfeature.